docs/modules/release-notes/pages/2023/v-3-26-jan-07-2023.adoc

```diff
@@ -0,0 +1,42 @@
+= v3.26 | January 7, 2023 (Legacy)
+:navtitle: v3.26 | January 7, 2023 (Legacy)
+
+These release notes cover Legacy Kobiton 3.26. As a reminder, in our November release, we started using 4.0+ as the numbering for releases of the new Kobiton, while Legacy Kobiton will continue with numbering starting with 3.25 and above. If you are using the new Kobiton, see the v4.1 release notes.
+
+If you have any questions or would like to update your Kobiton system, please send us an email at support@kobiton.com or contact your Account Executive.
+
+== New iOS and Android support
+
+Legacy Kobiton 3.26 supports the following:
+
+[cols="1,3"]
+|===
+|Hardware/Software |Full Support?
+
+|Pixel 7
+|Only manual testing
+
+|iOS 16.2
+|Everything except changing timezones
+
+|iOS 16.1.2
+|Everything except changing timezones
+
+|iOS 16.1.1
+|&#10003;
+
+|iOS 15.7.1
+|&#10003;
+|===
+
+== Webhooks for device state
+
+Admins can now setup webhooks to receive automatic updates of device status changes for Private and Local devices, such as when a device goes offline or online.
+//  TO DO: Learn more here (Need to make the webhook document)
```

Comment:
Let's take this in-page comment out and instead create a github issue to keep track of this. Other than that everything looks great!

Create issue here: https://github.com/kobiton/documentation/issues/new/choose